include/micm/configure/solver_config.hpp

```diff
@@ -119,7 +119,7 @@ namespace micm
 
   class JsonReaderPolicy
```

Comment:
```suggestion
  class ConfigReaderPolicy
```

Maybe we should rename the class to reflect the changes?